pkg/kubectl/describe/versioned/describe.go

```diff
@@ -2070,8 +2070,9 @@ func describeJob(job *batchv1.Job, events *corev1.EventList) (string, error) {
 		w := NewPrefixWriter(out)
 		w.Write(LEVEL_0, "Name:\t%s\n", job.Name)
 		w.Write(LEVEL_0, "Namespace:\t%s\n", job.Namespace)
-		selector, _ := metav1.LabelSelectorAsSelector(job.Spec.Selector)
-		w.Write(LEVEL_0, "Selector:\t%s\n", selector)
+		if selector, err := metav1.LabelSelectorAsSelector(job.Spec.Selector); err == nil {
```

Comment:
What sort of errors does that return? How bad would it be to actually print the error rather than just skip? Something like:
```
if err != nil {
      w.Write(LEVEL_0, "Selector:\tFailed to get selector: %s\n", err)
}
```